src/index.js

```diff
@@ -128,6 +130,8 @@ export const Utils = {
     StreamMessageValidator,
     CachingStreamMessageValidator,
     SigningUtil,
+    fetchTrackers,
```

Comment:
As both explorer and broker will use this in the same way:

```js
import * as Trackers from '../../contracts/TrackerRegistryDev.json'

const trackerList = await fetchTrackers(address, Trackers, jsonRpcProvider)
const trackersRing = new TrackersRing(['ws://1.1.1.1', 'ws://1.1.1.2'])
const trackerAddress = trackersRing.get('stream-1::0')
```

I would propose encapsulating this logic in a class and exporting it here, e.g.;

```js
class TrackerRegistry {
    constructor(address, jsonRpcProvider) { ... }

    getTracker(streamId) { ... }

    getAllTrackers() { ... }
}
```

So e.g. in client
```js
const registry = new TrackerRegistry("0xFFFFFFFFF", 'http://localhost:8545')
connect(registry.getTracker('streamId'))
```
or
```js
const registry = new TrackerRegistry("0xFFFFFFFFF", 'http://localhost:8545')
registry.getTrackers().forEach((trackerIp) =>connect(trackerIp))
```

Reply:
Good point, I'll update.